Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -4029,3 +4047,48 @@ def update_integration_context(context, object_keys=None, sync=True):
 
 class DemistoException(Exception):
     pass
+
+
+class ObjectMirror(object):
+    def __init__(self, object, entries):
+        self.object = object
+        self.entries = entries
+
+    def extract_for_local(self):
+        if self.object:
+            demisto.info(f'Updating object {self.object["id"]}')
+            return [self.object] + self.entries
+
+
+class SchemeMappingField:
+    def __init__(self, name, description=''):
+        self.name = name
+        self.description = description
+
+
+class SchemeTypeMapping:
+    def __init__(self, type_name='', fields=None):
+        self.type_name = type_name
+        self.fields = fields if fields else []
```

Comment:
support also if user provides just simple dictionary of keys and values of field name and field description

Reply:
in that case we don't really need this object, are you sure it's needed?

Reply:
You can remove it. And just have a function that adds new type to SchemaTypes